sycl/include/CL/sycl/detail/image_impl.hpp

```diff
@@ -239,7 +239,10 @@ class image_impl final : public SYCLMemObjT<AllocatorT> {
     MType = detail::convertChannelType(Format.image_channel_data_type);
     MNumChannels = getImageNumberChannels(MOrder);
 
-    getImageInfo(PI_IMAGE_INFO_ELEMENT_SIZE, MElementSize);
+    size_t ElementSize;
+    getImageInfo(PI_IMAGE_INFO_ELEMENT_SIZE, ElementSize);
+    assert(ElementSize <= 16);
+    MElementSize = static_cast<uint8_t>(ElementSize);
```

Comment:
Maybe it's better to change the type of MElementSize to size_t?

Reply:
Looks like a workaround.

Reply:
Fixed.